anesthetic/samples.py

```diff
@@ -523,6 +523,10 @@ def set_beta(self, beta, inplace=False):
             data.beta = beta
             return data
 
+    def convert_to_prior(self, inplace=False):
```

Comment:
The name of this function is a bit of a mouthful. Would there be anything wrong with renaming it to just `prior`, so that e.g. `samples.prior()` returned a prior-weighted set of samples?

Reply:
I avoided this name because of the discussion in #118, but there is nothing wrong with it. I have renamed it now.